<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BillMenu</class>
 <widget class="QWidget" name="BillMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QuickOrder</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1041</width>
     <height>461</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1021</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Selecione a mesa do consumidor</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="tableSelection"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Consumidores cadastrados</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="costumerSelection"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Voltar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="orderWidget">
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragOnly</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="showGrid">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>true</bool>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Id</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Prato</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Quantidade</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Preço</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Observação</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeButton">
         <property name="text">
          <string>Remover ordem</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="billButton">
         <property name="text">
          <string>Gerar conta</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
